src/thumbnailer.cc

```diff
@@ -60,9 +59,15 @@ Thumbnailer::Status Thumbnailer::GenerateAnimationNoBudget(
 
   // Fill the animation.
   for (auto& frame : frames_) {
-    if (!WebPAnimEncoderAdd(enc_, &frame.pic, frame.timestamp_ms, &config_)) {
+    WebPPicture new_pic;
+    WebPPictureInit(&new_pic);
```

Comment:
According to the documentation of WebPPictureCopy "The 'dst' picture need not be initialized as its content is overwritten."
So I think the call to WebPPictureInit is unnecessary (here and in other places in this file)